README.md

```diff
@@ -4,31 +4,48 @@ Gitflow basics for interns
 <h3>How to Gitflow</h3>
 
 **_Create Git Repository:_**
+	(Github)
 	1. Login to github
 	2. Create a new repository by clicking `New Repository`
 	3. Fill out the necessary details of your repository
 	4. Click on create repository
 
 **_Create a Develop Branch:_**
+	(Terminal)
 	1. Clone the repository
-	2. Initialize the repository
-	3. `git checkout -b Develop`
+	2. Go to the directory of the cloned repository
+	`Note: By default, the master branch will be currently selected`
+	3. `git checkout -b develop` to create the develop branch
+	Note: The newly created branch `develop` will become the selected branch
+	4. `git push origin develop` to put the develop branch in the remote repository
 
 **_Create New Issue for Each Task:_**
+	(Github)
 	1. Create issue by going to `Issues` and click `New Issue`
 	2. Fill out the necessary details about the issue
 
 **_Create New Branch for Each Issue:_**
-	1. `git checkout -b <task-name-issue#>`
+	(Terminal)
+	1. From the develop branch, create a new branch using: `git checkout -b <branch-name>`
+	`Note: Branch name would usually go like: <task-name-issue#>`
 	2. `git push -u origin <branch-name>`
-	3. Perform the task
-	4. Add, commit, and push the task to the branch
-	5. Pull request and merge branch to the `Develop` branch
-	6. Delete the repository branch of the finished task
-	7. `git branch -d <branch-name>` to delete the local branch
-	8. `git checkout develop` and `git pull` to update the develop branch
+	3. Complete the said task
+	4. `git add .` to add the updated set of files
+	5. `git commit -m "<task-description-fix-issue#>"` to save changes
+	6. `git push origin <branch-name>`
+
+**_Update the Develop Branch:_**
+	(Github)
+	1. Create a pull request to compare the develop branch and the task-branch
+	2. Merge task-branch to the `Develop` branch
```

Comment:
When can we merge the branch (what criteria) and who can merge the task branch to develop?